owncloudDomain/src/main/java/com/owncloud/android/domain/files/usecases/MoveFileUseCase.kt

```diff
@@ -66,6 +75,14 @@ class MoveFileUseCase(
         }
     }
 
+    private fun setLastUsageFile(file: OCFile, isAvailableLocally: Boolean ){
+        setLastUsageFileUseCase(SetLastUsageFileUseCase.Params(
+            fileId = file.id!!,
```

Comment:
We should not add dose '!!' because it's a source of future crashes.

Reply:
Done